src/activities/imageRecognition/ImageRecognition.jsx

```diff
@@ -190,6 +208,17 @@ module.exports = class ImageRecognition extends React.Component {
       }
       {
         this.state.activityState === ActivityState.Playing &&
+        <div>
+        <Row>
+          <Col style={{textAlign: 'center'}} xs={12}>
+            <PredictionUpload predictClass={(img) => {
+                        this.simpleTrainer.predict(img).then((result) => {
+                          this.setState({
+                            trainingResult: result
```

Comment:
[super nit, but possibly useful javascript tidbit] if you rename `result` to `trainingResult`, then you can simplify this line to:

```
this.setState({trainingResult})
```

([source](https://alligator.io/js/object-property-shorthand-es6/))